contracts/WildcardSteward_v3.sol

```diff
@@ -150,7 +151,7 @@ contract WildcardSteward_v3 is Initializable {
         require(
             !(deposit[tokenPatron] == 0 &&
                 totalPatronOwnedTokenCost[tokenPatron] > 0),
-            "You already own tokens but have no deposit"
+            "no deposit existing tokens"
```

Comment:
I shortened most of the revert reason strings. They don't need to be verbose. I have went with the standard of all lower case (I've just seen that in quite a few other solidity repos, not sure why)

Reply:
Great, any chance we can have a comma between so it makes more sense?

no deposit, own tokens